<launch>
  <!-- Launch blob detector with parameters -->
  <node pkg="simple_detect" name="blob_detect" type="backpack_blob_detect.py" output="screen">
   <rosparam command="load" file="$(find simple_detect)/config/backpack_blob_params.yaml"/>
 </node>

</launch>